packages/experimental/Button/SPEC.md

```diff
@@ -0,0 +1,191 @@
+# Button
+
+## Background
+
+The `Button` component enables users to trigger an action or event, such as submitting a form, opening a dialog, canceling an action, or performing a delete operation.
+
+## Sample Code
+
+Basic examples:
+
+```jsx
+<Button>Text</Button>
+<Button icon={<SVGIcon />} />
+<Button icon={<SVGIcon />}>Text</Button>
+<Button icon={<SVGIcon />} iconPosition="after">Text</Button>
+<Button appearance="primary">Text</Button>
+<Button disabled>Text</Button>
+<Button size="small">Text</Button>
+<Button size="large">Text</Button>
+```
+
+More examples in the Tester App.
+
+## Variants
+
+### Appearance
+
+The `Button` component has several apparance variants depending on where it's being used:
+
+- The default `Button` is rendered with its default styling indicating a trigger for an action.
+- appearance="primary": The `Button` is styled to emphasize that it represents the primary action.
+- appearance="subtle": The `Button` is styled to blend into its background to become less emphasized.
+- appearance="transparent": The `Button` is styled such that it has no background or border styling and is just emphasized through its content styling.
+
+### Icon
+
+The `Button` component can include an `icon` that appears before or after its `children`. If an `icon` is provided without any `children`, then the `Button` becomes an icon-only `Button`.
+
+### Shape
+
+- shape="rounded": The button as rounded corners. This is the default is shape is not set.
+- shape="circular": The button has completely round corners. A button of equal width and height will be a circle.
+- shape="square": The button has right-angle corners.
```

Comment:
Is this coming from web? Because I feel this can easily be accomplished with varying the cornerRadius / width

Reply:
Yeah this is one of those ease-of-use things.